rediscluster/client.py

```diff
@@ -736,6 +745,33 @@ def renamenx(self, src, dst):
 
         return False
 
+
+    def pubsub_channels(self, pattern='*', aggregate=True):
```

Comment:
I am not convinced that in the method `pubsub_channels` you have the defined keyword arguments `pattern` and `aggregate` and no `*args and **kwargs`. But in the method `pubsub_numsub` you have both and they all use/have the option to input `aggregate` as a kwarg. Please be consistent to use either or solution.